todo2.js

```diff
@@ -0,0 +1,82 @@
+var tasksArray = [];
+
+document.addEventListener("DOMContentLoaded", function () {
+  let taskInput = document.getElementById("input");
+  let taskList = document.getElementById("tasks-list");
```

Comment:
```suggestion
  const taskList = document.getElementById("tasks-list");
```

By default try to use const. Only use let if you know that the variable is supposed to change.